main/src/com/google/refine/grel/Parser.java

```diff
@@ -248,7 +248,9 @@ protected Evaluable parseFactor() throws ParsingException {
         }
 
         while (_token != null) {
-            if (_token.type == TokenType.Operator && _token.text.equals(".")) {
+        	if (_token.type == TokenType.Error) {
```

Comment:
This change is for checking the error: ```jso{nize(cells["URN"].value)```

Reply:
Perhaps it is worth adding a test for this in `GrelTests`, since it is independent from the templating exporter?